horreum-web/.prettierrc

```diff
@@ -3,6 +3,7 @@
   "trailingComma": "es5",
   "singleQuote": false,
   "printWidth": 120,
-  "tabWidth": 4,
-  "arrowParens": "avoid"  
+  "tabWidth": 2,
```

Comment:
are we using `.prettierrc` for anything else?  will changing the existing configuration break something else